main.css

```diff
@@ -0,0 +1,918 @@
+    align-items: center;
+    
+}
+
+.ftr-input {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.ftr-input-title {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 22px;
+    color: #141516;
+    margin-top: 0;
+    margin-bottom: 20px;
+}
+
+.ftr-input-txt {
+    color: #7C7C7C;
+    font-family: 'Mulish', sans-serif;
+    font-size: 16px;
+    line-height: 150%;
+    font-weight: 400;
+    border: 1px solid #afafaf;
+    box-sizing: border-box;
+    border-radius: 13px;
+    min-width: 480px;
+    padding: 13px 18px 13px 18px;
+    margin-bottom: 18px;
+}
+
+.ftr-input-msg {
+    color: #7C7C7C;
+    font-family: 'Mulish', sans-serif;
+    color: #141516;
+    font-size: 16px;
+    line-height: 150%;
+    font-weight: 400;
+    border: 1px solid #afafaf;
+    box-sizing: border-box;
+    border-radius: 13px;
+    min-width: 480px;
+    min-height: 94px;
+    padding: 13px 18px 83px 18px;
+    margin-bottom: 18px;
+
+}
+
+.ftr-input-btn {
+    background: #006AFF;
+    border-radius: 25px;
+    border: 0;
+    cursor: pointer;
+    padding: 13px 39px 13px 39px;
+    font-family: 'Mulish', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    color: white;
+}
+
+.afterthought {
+    background-color: #F5F5F5;
+}
+
+.aftertxt {
+    text-align: center;
+    font-family: 'Mulish', sans-serif;
+    font-size: 16px;
+    line-height: 150%;
+    font-weight: 400;
+    color: #141516;
+    margin-bottom: 10px;
+}
+body {
+    margin: 0;
+    padding: 0;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+/* a , a:visited {
+    text-decoration: none;
+    color: inherit;
+} */
+
+.hero-section {
+    background-color: #141516;
+    justify-content: center;
+    display: flex;
+
+}
+
+.hero-flex {
+    width: 1140px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+}
+
+.hero-nav {
+    padding-top: 35px;
+    display: flex;
+    justify-content: space-between;
+    font-family: 'Mulish', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    color: white;
+    margin-bottom: 103px;
+}
+
+.logo-box {
+    padding-right: 96px;
+}
+
+.hero-logo {
+    cursor: pointer;
+}
+
+.nav-left-ct {
+    display: flex;
+    flex-direction: row;
+    justify-content: center; 
+}
+
+.nav-right-ct {
+    display: flex;
+    flex-direction: row;
+}
+
+.nav-ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    min-width: 340px;
+
+}
+
+.right-nav-ul {
+    display: flex;
+    flex-direction: row;
+}
+
+.nav-link {
+    margin: 0;
+    /* padding-left: 40px;
+    padding-right: 40px; */
+}
+
+.nav-btn {
+    background: #006AFF;
+    border-radius: 25px;
+    border: 0;
+    cursor: pointer;
+    padding: 13px 29px 13px 29px;
+    font-family: 'Mulish', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    color: white;
+    margin-top: -13px;
```

Comment:
to zapisz jako jeden margines, tylko zastanawiam się czemu ma ujemną wartość, raczej niezbyt często się używa ujemnych, już wiem, w nawigacji nad pozostałymi elementami powinna też być przestrzeń tak by nie trzeba było podciągać tego buttona do góry, wszystkie elementy powinny być wyśrodkowane w pionie ale przestrzeń w której się najdują powinna pozwalać wszystkim elementom na to by się mieściły